stream_alert/apps/_apps/README.rst

```diff
@@ -96,3 +96,66 @@ back on local system
 
   # scp to local host's current directory
   $ scp -i /path/to/<private-key>.pem ec2-user@public.dns.address:~/pip_temp/pip.zip .
+
+Build dependencies on an virtual machine locally (python3.7)
+============================================================
+
+There is a Vagrantfile located in streamalert repo root directory. We can use this Vagrantfile to spin up an Ubuntu 18.04 virtual machine locally to build dependencies for box sdk or aliyun sdk.
+
+Download and install Vagrant if you haven't
++++++++++++++++++++++++++++++++++++++++++++
+
+Please visit `Vagrant download page <https://www.vagrantup.com/downloads.html>`_ for the information (recommend to install the latest Vagrant package).
+
+Bring up the virtual machine
+++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+  # It will take few mins to spin up virtual machine depending on your network speed
```

Comment:
suggestion:

`# It may take few minutes to start the virtual machine for the first time, depending on your network speed`